packages/node-helper/src/index.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+const {
+  parse32F,
+  parse64F,
+  parse32I,
+  parse64I,
+  parseU32,
+  isNanLiteral,
+  isInfLiteral
+} = require("./number-literals");
+
+import { longNumberLiteral, floatLiteral, numberLiteral } from "./nodes";
```

Comment:
Now I understand why you need https://github.com/xtuc/webassemblyjs/pull/404/files#r200809128.

Duplicated `nodes.js` to avoid the cyclic dep is not ideal IMO

Reply:
Ahaha.. this is `auto-generated` right? I think some refactoring needed.

Reply:
actually it depends on it. I think I have to redirect things here. But it is everywhere. I will find sometime this week

Reply:
ok thanks!